js/placeMessages.mjs

```diff
@@ -0,0 +1,196 @@
+                                                                                                    <p>${commentBody}</p>
+                                                                                                  </li>`
+                                                                                       }).reverse().join('') : ''}
+                        </ul>
+                      </div>`
+
+  messageLi.innerHTML += messageInfo
+
+  placementLocation.appendChild(messageLi)
+
+  const delMessageBtn = placementLocation.lastChild.querySelector('.remove-message')
+  if ( delMessageBtn ) {
+    delMessageBtn.addEventListener('click', () => deletePost(id))
+  }
+
+  const editMessageBtn = placementLocation.lastChild.querySelector('.edit-message')
+  const dialog = document.getElementById(`${id}-edit-modal`)
+  const imageRadio = document.getElementById(`${id}-send-image`)
+  const messageRadio = document.getElementById(`${id}-send-message`)
+  const sendImgForm = document.getElementById(`${id}-send-image-form`)
+  const sendMessageForm = document.getElementById(`${id}-send-message-form`)
+  const postTitle = document.getElementById(`${id}-post-title`)
+  const postTags = document.getElementById(`${id}-tags`)
+  const postMessage = document.getElementById(`${id}-message-text`)
+  const postImg = document.getElementById(`${id}-send-image`)
+  const postAltText = document.getElementById(`${id}-alt-text`)
+  if ( editMessageBtn ) {
+    editMessageBtn.addEventListener('click', () => {
+      dialog.showModal()
+      postTitle.value = title
+      postTags.value = tags.join(' ')
+      if (media) {
+        if (!imageRadio.hasAttribute('checked')) {
+          imageRadio.checked = true
+          messageRadio.checked = false
+        }
+        if (sendImgForm.classList.contains('hide-me')) {
+          sendImgForm.classList.remove('hide-me')
+        }
+        if (!sendMessageForm.classList.contains('hide-me')) {
+          sendMessageForm.classList.add('hide-me')
+        }
+        postImg.value = media
+        postAltText.value = body
+      } else {
+        if (!messageRadio.hasAttribute('checked')) {
+          imageRadio.checked = false
+          messageRadio.checked = true
+        }
+        if (sendMessageForm.classList.contains('hide-me')) {
+          sendMessageForm.classList.remove('hide-me')
+        }
+        if (!sendImgForm.classList.contains('hide-me')) {
+          sendImgForm.classList.add('hide-me')
+        }
+        postMessage.textContent = body
+      }
+    })
+  }
+
+  const dialogForm = document.getElementById(`${id}-edit-post`)
+  const editMessageType = document.getElementsByName(`${id}-messageOrImage`)
+  editMessageType.forEach(radioElement => {
+    radioElement.addEventListener('click', (ev) => postFormSelectType(ev.target))
+  })
+
+  dialogForm.addEventListener('submit', (ev) => {
```

Comment:
it is probably do to the id only been defined on the logged in users posts. a possible fix might be to wrap this eventListner in a if statment to check for null and undefined